spec/system/admin/enterprises/dfc_permissions_spec.rb

```diff
@@ -5,6 +5,10 @@
 RSpec.describe "DFC Permissions", feature: "cqcm-dev", vcr: true do
   let(:enterprise) { create(:enterprise) }
 
+  before do
+    skip "Puffing Billy seems to make our rspec processes hand at the end."
```

Comment:
Took me a moment to work out what this was saying; this should help others in future.
```suggestion
    skip "Puffing Billy seems to make our rspec processes hang at the end."
```